src/CoreBundle/private/rendering/layouts/includes/cms_head_data.inc.php

```diff
@@ -32,10 +34,9 @@
 <script src="<?= TGlobal::GetStaticURLToWebLib('/bootstrap/js/bootstrap.bundle.min.js?v4.1.3'); ?>" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
 <script src="<?= TGlobal::GetStaticURLToWebLib('/components/bootstrap3-typeahead/bootstrap3-typeahead.min.js'); ?>" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
 <script src="<?= TGlobal::GetStaticURLToWebLib('/javascript/jquery/jquery-form-4.2.2/jquery.form.min.js'); ?>" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
-<script src="<?= TGlobal::GetStaticURLToWebLib('/javascript/cms.js'); ?>" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
 <?php // Scrollbar JS needs to be loaded before CoreUI?>
 <script src="<?= TGlobal::GetPathTheme(); ?>/coreui/js/perfect-scrollbar.min.js" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
 <script src="<?= TGlobal::GetPathTheme(); ?>/coreui/js/coreui.min.js" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
 <script src="<?= TGlobal::GetPathTheme(); ?>/coreui/js/coreui-utilities.min.js" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->
 </head>
-<body class="<?=$cssClasses; ?>" <?=$bodyAttributes; ?>>
\ No newline at end of file
+<body class="<?=$cssClasses; ?>" <?=$bodyAttributes; ?>
```

Comment:
i think, the body close tag ">" must be still there?

Reply:
Indeed. Good catch. (added)